src/osvr/Common/ImagingComponent.cpp

```diff
@@ -71,9 +67,7 @@ namespace common {
 
             template <typename T>
             void allocateBuffer(T &, size_t bytes, std::true_type const &) {
-                m_imgBuf.reset(reinterpret_cast<OSVR_ImageBufferElement *>(
-                                   cv::fastMalloc(bytes)),
-                               &cv::fastFree);
+                m_imgBuf.reset(reinterpret_cast<OSVR_ImageBufferElement *>(malloc(bytes)));
```

Comment:
The reason that OpenCV was used here was to ensure that allocations would be appropriately aligned for optimized OpenCV operations that work on aligned buffers. For a non-OpenCV build of the core, these should at least be replaced with an aligned allocation - pretty sure I've got headers to do that in there.